diff --git a/src/basic/user-util.h b/src/basic/user-util.h
index fde43d6c1..b148fc745 100644
--- a/src/basic/user-util.h
+++ b/src/basic/user-util.h
@@ -89,8 +89,8 @@ int take_etc_passwd_lock(const char *root);
 #define UID_INVALID ((uid_t) -1)
 #define GID_INVALID ((gid_t) -1)
 
-#define UID_NOBODY ((uid_t) 65534U)
-#define GID_NOBODY ((gid_t) 65534U)
+#define UID_NOBODY ((uid_t) 99U)
+#define GID_NOBODY ((gid_t) 99U)
 
 /* If REMOUNT_IDMAPPING_HOST_ROOT is set for remount_idmap() we'll include a mapping here that maps the host
  * root user accessing the idmapped mount to the this user ID on the backing fs. This is the last valid UID in
